.github/workflows/conformance-clustermesh.yaml

```diff
@@ -730,10 +730,10 @@ jobs:
   merge-upload-and-status:
     name: Merge Upload and Status
     if: ${{ always() }}
-    needs: installation-and-connectivity
+    needs: [wait-for-images, installation-and-connectivity]
     uses: ./.github/workflows/common-post-jobs.yaml
     secrets: inherit
     with:
       context-ref: ${{ inputs.context-ref || github.sha }}
       sha: ${{ inputs.SHA || github.sha }}
-      result: ${{ needs.installation-and-connectivity.result }}
+      result: ${{ needs.wait-for-images.result == 'success' && needs.installation-and-connectivity.result == 'success' && 'success' || 'failure' }}
```

Comment:
Is it actually necessary to make these workflows depend on `wait-for-images`? I think that `needs.installation-and-connectivity.result == 'success' && 'success' || 'failure'` should be enough, given that it would report a failure in case the workflow gets skipped. Not that it changes much, but it looks a bit simpler.